Update documentation (#96)
ReachFive/identity-android-sdk: README.md

Since the demo application uses Google services, you need to create a new [Firebase](https://firebase.google.com/) project.
Download the `google-services.json` file associated and put it at the root of the `/app` directory.

> You also need to set the ReachFive client configuration in the `/app/src/main/assets/env` file as below (you'll need to create before the `assets` folder):

Reviewer: ```suggestion
You also need to set the ReachFive client configuration in the `/app/src/main/assets/env` file as below (create the `assets` folder if necessary):
```
